app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -161,13 +160,23 @@ class PipelineSampleReport extends React.Component {
     const attr = className.split(' ');
     const taxId = attr[2];
     $(`.report-row-species.${taxId}`).toggleClass('hidden');
-    // HACK.  Flipping plus/minus should be done with React, not DOM change.
-    if (attr[1] == 'fa-angle-right') {
-      e.target.attributes.class.nodeValue = `fa fa-angle-down ${taxId}`;
-    } else {
-      e.target.attributes.class.nodeValue = `fa fa-angle-right ${taxId}`;
-    }
-    e.nativeEvent.stopImmediatePropagation();
+    $(`.report-arrow.${taxId}`).toggleClass('hidden');
+  }
+
+  expandTable(e) {
```

Comment:
Note that the negative genera ("all taxa without genera" and "all artificial constructs") will not be expanded by the Table expander.    I thought this is how users would want it.   Let me know if I am wrong.